obelix/__init__.py

```diff
@@ -231,11 +234,27 @@ def download_data(self, output_path, commit_id=None, local=False):
         df.to_csv(output_path / "LiIonDatabase.csv")
     
     def read_data(self, data_path, no_cifs=False):
```

Comment:
Add an argument to choose to remove room temperatue only and make it true by default